apps/components-e2e/src/components/sunburst-chart/sunburst-chart.e2e.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { resetWindowSizeToDefault, waitForAngular } from '../../utils';
+import {
+  body,
+  centralLabel,
+  centralValue,
+  overlay,
+  segments,
+  sliceShephard,
+  valueJack,
+  percentBtn,
+} from './sunburst-chart.po';
+
+// Reduced speed of hovering should get our e2e tests stable.
+// We should think about removing the dtOverlay and using the cdk one,
+// that is not flaky on other e2e tests #86
+const hover: MouseActionOptions = {
+  // Slowest possible speed should help as workaround til the issue is fixed.
+  // The issue #646 is opened for this.
+  speed: 0.01,
```

Comment:
Hmm, this feels like a not very representative speed. 
Although this will keep the tests "non-flaky", I'm not entirely sure if it helps us. Actual users are probably faster with hovering. Can we use speed of like 0.6 and still achieve stability.